### PR TITLE
chore(react-query): update JSDoc for `useSuspenseQueries`, `useSuspenseInfiniteQuery`

### PR DESCRIPTION
**Pull request by gwansikk**
# Overview

Updates the JSDoc for the `useSuspenseQueries` and `useSuspenseInfiniteQuery` functions.

## PR Checklist

- [x] I did below actions if need

1. I read the [Contributing Guide](https://github.com/toss/suspensive/blob/main/CONTRIBUTING.md)
2. I added documents and tests.

### CODE REVIEW

File: packages/react-query/src/useSuspenseInfiniteQuery.ts [hunk 1]
```diff
@@ -27,7 +27,7 @@ export interface UseSuspenseInfiniteQueryOptions<
   > {}
 
 /**
- * This hook is wrapping useInfiniteQuery of @tanstack/react-query v4 with default suspense option.
+ * This hook is wrapping useInfiniteQuery of `@tanstack/react-query` v4 with default suspense option.
```
Comment (gwansikk) [inline packages/react-query/src/useSuspenseInfiniteQuery.ts:30]: I've made revisions for consistency.

File: packages/react-query/src/useSuspenseQueries.ts [hunk 2]
```diff
@@ -109,6 +109,10 @@ export type SuspenseQueriesResults<
           : // Fallback
             Array<UseSuspenseQueryResult>
 
+/**
+ * This hook is wrapping useQueries of `@tanstack/react-query` v4 with default suspense option.
```
Comment (gwansikk) [inline packages/react-query/src/useSuspenseQueries.ts:113]: I added the JSDoc for this function as it was missing.

File: packages/react-query/src/useSuspenseInfiniteQuery.ts (hunk 1 above)
Comment (manudeli) [reply packages/react-query/src/useSuspenseInfiniteQuery.ts:30]: Thanks! So much better jsdoc 👍

### AS-IS
image

### TO BE
image

File: packages/react-query/src/useSuspenseQueries.ts (hunk 2 above)
Comment (manudeli) [reply packages/react-query/src/useSuspenseQueries.ts:113]: 👍

**Review by manudeli — APPROVED** [review]

**Comment by manudeli** [comment]
Thanks! You made so much better jsdoc!

**Pull request merged**